coding/code-formatting.md

```diff
@@ -0,0 +1,9 @@
+# Code formatting
```

Comment:
This seems completely JavaScript specific isn't it? Should it be coding/javascript-code-formatting.md or something?

Reply:
Or at least under a `# JavaScript` heading.

Reply:
Prettier is not JS specific and will format pretty much all of the languages we use.

<img width="916" alt="screen shot 2018-07-17 at 14 28 19" src="https://user-images.githubusercontent.com/505570/42820317-c28adf50-89cd-11e8-99ef-6d956533dd93.png">

Reply:
@barrymcgee importantly that list doesn't include Python, also Ruby, Go, Bash ...

A `# Front-end` heading would make sense. But the AirBnB stuff and ESLint are *definitely* JS specific. So that should be under a `#JavaScript` heading of some sort.

Reply:
Interesting to see if the Scss styling rules match our rules in this practises

Reply:
@barrymcgee Yep fair enough, I like the new structure. Thanks for updating that.